ui/perfherder/compare/CompareTable.jsx

```diff
@@ -172,7 +176,10 @@ export default class CompareTable extends React.Component {
         <thead>
           <tr className="subtest-header bg-lightgray">
             <th className="text-left">
-              <span>{testName}</span>
+              <span>
+                {testName}
+                {framework === 'talos' && <a href={url}> (docs)</a>}
```

Comment:
Could you look for a way to display the `(docs)` link with blue, like normal anchor links look like?